core/drand_beacon_control.go

```diff
@@ -767,7 +767,7 @@ func (bp *BeaconProcess) extractGroup(old *drand.GroupInfo) (oldGroup *key.Group
 		// try to get the current group
 		if bp.group == nil {
 			bp.state.Unlock()
-			return nil, errors.New("drand: can't init-reshare if no old group provided")
+			return nil, errors.New("can't init-reshare if no old group provided - try using the --from flag")
```

Comment:
I think we should try and keep flags out of the library part of drand as much as possible...
Can't we catch this earlier on, on the client side? Maybe by handling this error? (Make it a named error instead.)

Reply:
Right now we can't get a `BeaconProcess` from the cli, just use the `ControlClient` - we could expose something to enable this potentially?  e.g. `IsMember(beaconId)`.  I fiddled a bit with `ctrlClient.LoadBeacon(beaconID)` to in the cmd to see if it would give me an indication of whether the network has been joined but it didn't seem to

Reply:
Have updated the advisory text to be meaningful without a flag